### DISCUSSION TITLE
Token scope to use for removing collaborators?

### DISCUSSION THREAD

**Discussion by nsm10** [community/community#24318] [Apps, API and Webhooks]
<p>Scope for Personal access token with the requirement of removing collaborators from private GitHub repo</p>

**Comment by byrneh** [comment, marked as answer]
<p>I succesfully checked and deleted and collaborator on a Private repository with token scoped “repo” (Full control of private repositories)<br>
The REST api  <a href="https://docs.github.com/en/rest/reference/orgs#list-app-installations-for-an-organization" rel="noopener nofollow ugc">remove-a-repository-collaborator</a> returned “status 204 No Content”, when successful<br>
With an incorrectly scoped permission token it returns 404 Not Found when attempting check or delete collaborator</p>

**Comment by nsm10** [comment]
<p>Yes I did read up scope doc from GitHub. Here is the <a href="https://docs.github.com/en/developers/apps/scopes-for-oauth-apps" rel="noopener nofollow ugc">link</a> for anyone who has problems in understanding what scope to choose for their particular requirement</p>
